admin/src/components/UserTable.vue

```diff
@@ -72,6 +72,14 @@
         </b-button>
       </template>
 
+      <template #cell(has_elopage)="row">
+        <b-button :variant="row.item.hasElopage ? 'success' : 'danger'" size="md" class="mr-2">
+          <b-icon
+            :icon="row.item.hasElopage ? 'arrow-up-short' : 'exclamation-triangle-fill'"
```

Comment:
I do not like the headlines and the icons you use - why is "Details" the actual Creation?
Please rename buttons and change icons to be more expressive of their respective use.
If you do not change this in this PR, please make an Issue for this.